l3kernel/l3fp-functions.dtx

```diff
@@ -1,6 +1,6 @@
 % \iffalse
 %
-%% File l3fp-functions.dtx (C) Copyright 2012-2018,2020,2021 The LaTeX Project
+%% File l3fp-functions.dtx (C) Copyright 2012-2018,2020,2021,2023 The LaTeX Project
```

Comment:
not important but I wonder if you shouldn't simplify these year lists to just 2012-2023. It isn't that significant that 2022 there was no change :-)

Reply:
IANAL :)

Reply:
really? :-) had to look IANAL up first ... not in my vocabulary. But anyway, I think we have and had the (C) all the time even if we didn't explicitly mentioned it in some years because there was no change. But up to you.

Reply:
@FrankMittelbach  The 2e sources use a simple range, all of the L3 ones are current 'explicit change years only' - so a change here should be handled separately from the PR.

Reply:
sure fine. I'm not worried I just think it looks more tidy